Fix for D3P_OCTOPUS_Driver2208_UART_BLTouch_LCD_BTT_TFT Build

andrewseago opened the pull request:
Fixed typo in Configuration_Tenlog.h that had falsesirius fm instead of false. 

Removed Y_Min = -5 changes for OpticalY. 

Modified Marlin/src/pins/stm32f4/pins_BTT_OCTOPUS_V1_common.h to better match more recent versions of the file in order to resolve build issues with D3P_OCTOPUS_Driver2208_UART_BLTouch_LCD_BTT_TFT

andrewseago committed:
Updated OpticalY YMin in Configuration_Tenlog.h

andrewseago committed:
Fixed Typos in Configuration_tenlog.h and Octopus_V1_common.h

klack commented on Marlin/Configuration_tenlog.h:
```diff
@@ -218,6 +218,7 @@
   #undef  Y_MIN_ENDSTOP_INVERTING
   #define Y_MIN_ENDSTOP_INVERTING true
   #define LUX_ENDSTOP_Y_OFFSET -8
+  #define LUX_REFERENCE_Y_MIN_POS -5
```
Looks like it is still in here

klack commented on Marlin/Configuration_tenlog.h:
```diff
@@ -360,16 +361,17 @@
 #endif
-
-#define LUX_REFERENCE_Y_MIN_POS 0
+#ifndef LUX_REFERENCE_Y_MIN_POS
```
<-

klack commented on Marlin/Configuration_tenlog.h:
```diff
@@ -360,16 +361,17 @@
 #define X_MAX_POS LUX_MEASURED_X_BED_SIZE - LUX_REFERENCE_EXTRUDER_WIDTH
 
-#define Y_MIN_POS 0 //Always 0
+#define Y_MIN_POS LUX_REFERENCE_Y_MIN_POS //Always 0
```
<--

klack commented:
Nice description

andrewseago committed:
Update Configuration_tenlog.h

andrewseago commented:
Requested changes should be applied now.

klack commented on Marlin/Configuration_tenlog.h:
```diff
@@ -369,7 +368,7 @@
 #define X_MAX_POS LUX_MEASURED_X_BED_SIZE - LUX_REFERENCE_EXTRUDER_WIDTH
 
-#define Y_MIN_POS 0 //Always 0
+#define Y_MIN_POS LUX_REFERENCE_Y_MIN_POS //Always 0
```
On last thing, can you put this back to 0

andrewseago committed:
Configuration_tenlog.h:
Added Tenlog TL-D4 MachineTLD4, Tenlog TL-D6 MachineTLD6, Tenlog HANDS2 MachineTLD2P, and Tenlog HANDS2 Pro MachineTLH2P.
Moved LUX_REFERENCE_X2_MAX_POS to each machine type as it is different on each system.

platformio.ini and luxuri.ini:
Added D6P_2209_OpticalY build option to platformio.ini and luxuri.ini
